README.md

```diff
@@ -5,17 +5,21 @@ This repository contains the source code and assets for the Hello Engine, Hello
 Note that before you deploy, you must accept the [Qlik Core EULA](https://qlikcore.com/beta/) by setting the `ACCEPT_EULA` environment variable.
 
 ```sh
-$ ACCEPT_EULA=yes docker-compose up -d
+ACCEPT_EULA=yes docker-compose up -d
 ```
 
 ## Contents
 
 - [hello-engine](./src/hello-engine/) - Hello Engine example source code
 - [hello-data](./src/hello-data/) - Hello Data example source code
-- [hello-visualization](./src/hello-visualization/) - Example source code for Visualizations:- Line chart and Scatter plot.
+- [hello-visualization](./src/hello-visualization/) - Example source code for a line chart and a scatter plot visualization
 - [test](./test) - Function tests for the examples (bash scripts)
 - [data](./data) - The Movies data, used as user data in the examples
 
+## Screenshot
+
+<img src="./src/hello-visualization/resources/hello-viz.png" width="600">
```

Comment:
You can also use `![./src/hello-visualization/resources/hello-viz.png](./src/hello-visualization/resources/hello-viz.png)`, any specific reason to hardcode the width?